resources/wrapper/godelw

```diff
@@ -161,23 +164,35 @@ GODEL_BASE_DIR=${GODEL_HOME:-$HOME/.godel}
 # determine OS
 OS=""
 EXPECTED_CHECKSUM=""
-case "$(uname)" in
-    Darwin*)
+case "$(uname)-$(arch)" in
+    Darwin*-x86_64)
```

Comment:
Based on testing, I think it's safe to remove the `*` and just have this be `Darwin-x86_64`